provisioning/templates/dashboard.html.j2

```diff
@@ -215,15 +219,18 @@
         <thead>
           <tr>
             <th>Database name</th>
+            <th>Engine</th>
             {% if 'adminer' in installed_extras -%}
               <th></th>
             {%- endif %}
           </tr>
         </thead>
         <tbody>
-          {% for database in mysql_databases -%}
+          {% set database_list = (drupalvm_database == 'pgsql') | ternary(postgresql_databases, mysql_databases) %}
```

Comment:
`ternary` filters were added in Ansible v1.9. Does the project have a minimum version requirement set at the moment? At the moment it's v2 because of the galaxy bug I guess.